app/components/Button/index.tsx

```diff
@@ -8,20 +8,24 @@ type ButtonProps = {
   className?: string
   tooltip?: string
   icon?: ReactNode
+  leftIcon?: ReactNode
 }
-const Button = ({children, action, tooltip, icon, className}: ButtonProps) => {
+const Button = ({children, action, tooltip, icon, leftIcon, className}: ButtonProps) => {
   const classes = ['button', className, tooltip && 'tooltip'].filter((i) => i).join(' ')
   if (typeof action === 'string') {
     return (
       <Link to={action} className={classes} data-tooltip={tooltip}>
+        {leftIcon && leftIcon}
```

Comment:
I don't like how this is getting bloated. Why not just pass the icon in as a child, rather than having special parameters? So e.g.:
```
<Button>
   <SomeIcon />
   bla bla
</Button>
```
rather than
`<Button leftIcon={<SomeIcon/>}>bla bla</Button>`?